web/src/notifications.js

```diff
@@ -401,9 +401,12 @@ export function message_is_notifiable(message) {
         return true;
     }
 
-    // Messages to muted streams that don't mention us specifically
```

Comment:
This comment is not needed as the comment on line 398 already mentions this thing. (in a different way)